beam-postgres/Database/Beam/Postgres/Syntax.hs

```diff
@@ -1371,6 +1421,23 @@ pgDropExtensionSyntax extName =
 --                                        Nothing
 --      buildJoinFrom tbl newSource Nothing
 
+
+-- TODO
+-- Don't directly expose sum-type constructors?
```

Comment:
They should be exposed from the `Syntax` module, but they should not be exposed from the main `Database.Beam.Postgres` module, if that makes sense.